pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -200,7 +202,11 @@ func NewVolumeManager(
 type volumeManager struct {
 	// kubeClient is the kube API client used by DesiredStateOfWorldPopulator to
```

Comment:
The comment needs to be update for RPclient.

Reply:
fixed